### ISSUE TITLE
Support @viewport

### ISSUE THREAD

**Issue by yisibl** [postcss/autoprefixer#380]
```css
@-o-viewport {
  width: 320px;
}

@-ms-viewport {
  width: auto;
}
```

We need to increase the caniuse.com data.

* https://developer.mozilla.org/zh-CN/docs/Web/CSS/@viewport
* http://msdn.microsoft.com/en-us/library/ie/hh869615%28v=vs.85%29.aspx
* https://dev.opera.com/articles/an-introduction-to-meta-viewport-and-viewport/

**Comment by ai** [comment]
Easy to do, but we need PR for Can I Use

**Comment by cvrebert** [comment]
Hmm? It already seems to have the right prefixing info:
https://github.com/Fyrd/caniuse/blob/0bd55dabc824d823747aecebae68dce612d6cced/features-json/css-deviceadaptation.json

**Comment by ai** [comment]
@cvrebert wow, thanks! It is very useful!

**Comment by yisibl** [comment]
@ai This data is a bit of a problem, I will update it.

**Comment by ai** [comment]
Done https://github.com/postcss/autoprefixer-core/commit/d266a764e4aebb0edc7a1ca50c267aea39f03e21

**Comment by ai** [comment]
Released today in 5.1
